src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathsinglefilter/Example5.java

```diff
@@ -1,6 +1,7 @@
 /*xml
 <module name="Checker">
   <module name="TreeWalker">
+    <module name="com.puppycrawl.tools.checkstyle.checks.modifier.RedundantModifierCheck"/>
```

Comment:
please add Check to this collection
https://github.com/checkstyle/checkstyle/commit/30717c3384103d879d069b8e0b25e3dfbd8bd54e#diff-58cc197c3dfaae5245ca0fb29e5892861aeecff23be676ea61fc95f320b7c318R316

so you will not need fully qualified name here.
fully qualified name will confuse users.

Reply:
this should resolve https://github.com/checkstyle/checkstyle/pull/16282#issuecomment-2644441855